toolkit/scripts/build_tag_imagecustomizer.mk

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Azure Linux Image Customizer version.
+# This is using semantic versioning.
+#
+# IMAGE_CUSTOMIZER_VERSION should have the format:
+#
+#   <major>.<minor>.<patch>
+#
+# and should hold the value of the next (or current) official release, not the previous official
+# release.
+IMAGE_CUSTOMIZER_VERSION ?= 0.3.0
```

Comment:
Did you increment the version on purpose as part of the move?

Reply:
Yes.